structuredevent-model/src/main/java/com/sequenceiq/environment/parameter/dto/AzureParametersDto.java

```diff
@@ -4,37 +4,51 @@
 
     private AzureResourceGroupDto azureResourceGroupDto;
 
-    public AzureParametersDto(Builder builder) {
+    private AzureResourceEncryptionParametersDto azureResourceEncryptionParametersDto;
+
+    private AzureParametersDto(Builder builder) {
         azureResourceGroupDto = builder.azureResourceGroupDto;
+        azureResourceEncryptionParametersDto = builder.azureResourceEncryptionParametersDto;
```

Comment:
Please add unit tests for the changed parts of `AzureParametersDto` (at least test the builder & getter).